internal/pkg/mapper/configmetadata/local_save.go

```diff
@@ -0,0 +1,32 @@
+package configmetadata
+
+import (
+	"sort"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapBeforeLocalSave - store config metadata to manifest.
+func (m *configMetadataMapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe) error {
+	manifest, ok := recipe.ObjectManifest.(*model.ConfigManifest)
+	if !ok {
+		return nil
+	}
+
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	if config.Metadata != nil {
```

Comment:
Mohol by si spravit metodu, `config.MetadataOrderedMap()`.

Reply:
👍 jo, je to mnohem hezčí